docs_sources/api.md

```diff
@@ -147,6 +148,19 @@ Flag indicating whether the TPOT instance will reuse the population from previou
 Setting <em>warm_start</em>=True can be useful for running TPOT for a short time on a dataset, checking the results, then resuming the TPOT run from where it left off.
 </blockquote>
 
+<strong>memory</strong>: a sklearn.external.joblib.Memory object or string, optional (default=None)
+<blockquote>
+If supplied, TPOT uses memory caching while optimizing.
```

Comment:
Make it clear that this is pipeline caching and link to http://scikit-learn.org/stable/modules/pipeline.html#caching-transformers-avoid-repeated-computation